src/index.ts

```diff
@@ -21,6 +23,14 @@ export function convertAxeToSarif(axeResults: Axe.AxeResults): SarifLog {
     return sarifConverter.convert(decoratedAxeResults, {});
 }
 
+export function convertAxeToSarif21(axeResults: Axe.AxeResults): Sarif.Log {
```

Comment:
This has the result of publishing a new API surface in our package, which I think I'd rather not do quite yet; I'd rather update the existing API name with a breaking change/major version update when the time comes. For now, I think we should just have the integration test use the defaultSarifConverter21() call directly